hands-on-exercise/5-ibc-adv/4-ibc-app-steps.md

```diff
@@ -56,7 +56,6 @@ The code in this section was scaffolded with Ignite CLI v0.22.1. This version in
 <br/><br/>
 **It is thus highly discouraged to deploy any code in production using ibc-go code scaffolded by Ignite CLI v0.22.1.**
 <br/><br/>
-<!--- TODO: add link to section on upgrading ibc-go version --->
```

Comment:
You can add an empty line above and it will disappear.

Reply:
I do not think that we are going to have an "upgrading ibc-go version" chapter in the near feature and if we do an update, this section will be updated to a newer version anyway.